### PR TITLE
Global logback config

### PR DESCRIPTION
**Pull request by dst**

### CODE REVIEW

File: src/main/resources/logback.groovy [hunk 1]
```diff
@@ -7,17 +7,76 @@ import ch.qos.logback.core.status.OnConsoleStatusListener
...
+            return Integer.parseInt(maxHistory)
+        } else {
+            return DEFAULT_ROLLING_MAX_HISTORY
+        }
+    }
+
+    private File globalPropertiesFile() {
+        def env = System.getenv()
+        String configFolder = env['CONFIG_FOLDER']
+        return new File("${configFolder}/common/global.properties")
```
Comment (szpak) [inline src/main/resources/logback.groovy]: Will it work on Windows?

Comment (dst) [inline src/main/resources/logback.groovy]: Yes, it will. You asked a similar question in the past and I have already explained it to you :)

Comment (szpak) [inline src/main/resources/logback.groovy]: I don't remember. Maybe it wasn't me or maybe I have too many threads to handle I just threw away your explanations from my mind ;). Anyway great that you checked it.

**Comment by dst** [comment]
I'm thinking about moving LogbackConfig class to micro-infra-spring. WDYTAI?

**Comment by szpak** [comment]
There are no direct dependencies to Logback, so it sound sensible.

**Comment by marcingrzejszczak** [comment]
You would move it to which module?

**Comment by dst** [comment]
config, already done in https://github.com/4finance/micro-infra-spring/pull/330

**Comment by dst** [comment]
@marcingrzejszczak, I have used logback configuration from micro-infra-spring. Could you please look now?

**Comment by marcingrzejszczak** [comment]
+1

**Pull request merged**